src/main/java/virtual_pet/VirtualPet.java

```diff
@@ -0,0 +1,46 @@
+package virtual_pet;
+public class VirtualPet {
+
+    public String petName;
+    public int hunger;
+    public int thirst;
+    public int energy;
+
+    public VirtualPet(String petName, int hunger, int thirst, int energy) {
+
+        this.petName = petName;
+        this.hunger = hunger;
+        this.thirst = thirst;
+        this.energy = energy;
+    }
+    public void changeHunger(int h){
+        hunger += h;}
+    public void changeThirst(int t){
+        thirst += t;
+    }
+    public void changeEnergy(int e){
+        energy += e;
+    }
+    public String getPetName(){
+        return petName;
+    }
+    public int getHunger(){
+        return hunger;}
+    public int getThirst(){
+        return thirst;}
+    public int getEnergy(){
+        return energy;
+    }
+    public void setHunger(int hunger){
```

Comment:
Use of a setter is poor programming and not best practice.  And seeing what you were attempting to do in your tick() this is not needed and could have been handled in your tick.  Please refer to comment for tick().